app/src/components/surveys/SurveysList.tsx

```diff
@@ -77,12 +77,7 @@ const SurveysList: React.FC = () => {
                       {row.surveyData.survey_details.survey_name}
                     </Link>
                   </TableCell>
-                  <TableCell>
-                    {[
-                      ...row.surveyData.species.focal_species_names,
-                      ...row.surveyData.species.ancillary_species_names
-                    ].join(', ')}
-                  </TableCell>
+                  <TableCell>{row.surveyData.species.focal_species_names.join(', ')}</TableCell>
```

Comment:
Small extra change: Can we change the join to use a semi-colon. Noticing that when you have multiple species it is hard to separate each species since we also use commas within a species string. Maybe do the same anywhere else we concatenate species (might just be here and in the survey details?).

Reply:
done. Survey details has them broken down into typography components so they each has their own line